workspaces/testbed-workspace/scripts/install-developer.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/bash
+
+# SPDX-FileCopyrightText: 2020 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Install development software
+# Used to minimize the software installed in the base image (testbed-workspace)
+
+# Development tools
+dnf install -y python3 wget git jq vim zsh sudo psmisc bind-utils mlocate
```

Comment:
`wget` is already installed on the Dockerfile. Do we need it here?

Reply:
no, good catch. Earlier was not in the docker file. I'll merge the PR (to keep the approval) and remove wget